clients/wallet/src/keys.rs

```diff
@@ -0,0 +1,30 @@
+#[allow(unused_imports)]
+use dotenv::dotenv;
+use std::env;
+
+// Gets all environment variables contained in the local .env file and returns
+// the given variable by key if available. This will override the variables
+// previously passed to the environment.
+// A variable not .env but passed to the environment will not be overridden.
+fn get_env_variables(key: &str) -> Option<String> {
+    dotenv::from_filename("../.env").ok();
+    env::var(key).ok()
+}
+
+pub fn get_dest_secret_key_from_env(is_mainnet: bool)-> String{
+	let maybe_secret = match is_mainnet {
+		true => get_env_variables("DEST_SECRET_MAINNET").expect("Failed to read secret key from environment"),
+		false => get_env_variables("DEST_SECRET_TESTNET").expect("Failed to read secret key from environment"),
+	};
+
+	maybe_secret
+}
+
+pub fn get_source_secret_key_from_env(is_mainnet: bool)-> String{
+	let maybe_secret = match is_mainnet {
+		true => get_env_variables("SOURCE_SECRET_MAINNET").expect("Failed to read secret key from environment"),
```

Comment:
I know it's going to be long, but is it more readable to name the env variable
`SOURCE_`**_`STELLAR`_**`_SECRET_MAINNET`? just so we know that this secret key is for stellar.

@ebma @gianfra-t ?

Reply:
Sure, makes sense 👍 Let's call them `SOURCE_STELLAR_SECRET_MAINNET`, `SOURCE_STELLAR_SECRET_TESTNET`, `DEST_STELLAR_SECRET_MAINNET`, and `DEST_STELLAR_SECRET_TESTNET`

Reply:
@gianfra-t I now added the secrets using the naming in my previous comment as repository secrets to this Github repository. 

I did some googling and it seems like to use them we need to echo them into a local `.env` file, see [here](https://stackoverflow.com/questions/61117865/how-to-set-environment-variable-in-node-js-process-when-deploying-with-github-ac). So we'd need to edit the workflow file of the Github CI action and add something like
```
- name: create env file
      run: |
        touch ./clients/.env
        echo SOURCE_STELLAR_SECRET_MAINNET=${{ secrets.SOURCE_STELLAR_SECRET_MAINNET }} >> .env
        echo ...
```

Reply:
That would be to create the .env, but I think they can be passed to the environment of the job as:
```
steps:
  - name: Hello world action
    env: 
      super_secret: ${{ secrets.SuperSecret }}
 ```
 At least this is what I understood from [here](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions#using-secrets-in-a-workflow).

Reply:
If that works, even better 👍 I was not sure if cargo would pick them up or if the scope is somehow limited if we define them like this.

Reply:
It seems to work! Tests are failing now because of a `SelfPaymentError` which means that probably we need to shuffle `source` <-> `destination` somewhere, but the environment variables are read.

Reply:
Oh okay, thanks! indeed that was hard to guess.